test/azure/terraform_azure_sqlmanagedinstance_example_testlong.go

```diff
@@ -0,0 +1,63 @@
+// SQL Managed Instance takes 6-8 hours for deployment, Exclude this from the worflow by naming it as *_testlong.go
```

Comment:
I would rename the file to a be a valid test file 'sql_mangedinstace_test' and since get rid of all the tags with a minor change to the azure ci workflow to run tests based on the azure tag this would exclude the test from getting executed by using 'go test ./azure/*  --tages=azure -v -timeout 90m' https://github.com/gruntwork-io/terratest/blob/b8689cdaceb2cbdcbfa9e95e251fbd71e76c186b/.github/workflows/ci.yml#L134

Reply:
Thanks Hadwa updated it